bot/cogs/commands.py

```diff
@@ -101,27 +115,112 @@ async def on_message(self, message):
                 return
 
     @commands.command()
-    async def version(self, ctx):
+    async def version(self, ctx: commands.Context):
+        """Usage: `version`
+        Response: Fred's current version
+        Notes: Command is a useful is-alive check"""
         await self.bot.reply_to_msg(ctx.message, self.bot.version)
 
-    @commands.command()
-    async def help(self, ctx):
-        await self.bot.reply_to_msg(ctx.message, "Sorry, this command is temporarily unavailable")
+    @commands.group()
+    async def help(self, ctx: commands.Context) -> None:
+        """[Help Commands!](https://www.youtube.com/watch?v=2Q_ZzBGPdqE)
```

Comment:
This is repeated a lot (with every command). Can't you insert them instead of having them in plain text?

Reply:
This is for the help text for help commands themselves, there's no easy way (or at least one i want to implement) to add this only to them automatically.